drive_cli/utils.py

```diff
@@ -369,10 +369,23 @@ def upload_file(name, path, pid):
         'parents': [pid],
         'mimeType': file_mimeType
     }
-    media = MediaFileUpload(path, mimetype=file_mimeType)
-    new_file = service.files().create(body=file_metadata,
-                                      media_body=media,
-                                      fields='id').execute()
+    if os.stat(path).st_size <= 256 * 1024:
+        media = MediaFileUpload(path, mimetype=file_mimeType)
+        new_file = service.files().create(body=file_metadata,
+                                          media_body=media,
+                                          fields='id').execute()
+    else:
+        CHUNK_SIZE_MB = 1  # MB. You may want to increase the size to a higher speed if the network restrictions allow
```

Comment:
```suggestion
            CHUNK_SIZE_MB = int(os.getenv("CHUNK_SIZE_MB") or 1
```

Reply:
Thanks:) 
Please make both the changes (this + `chunksize` argument)